client/packages/system/src/Clinician/ClinicianSearchInput.tsx

```diff
@@ -59,7 +59,7 @@ export const ClinicianSearchInput: FC<ClinicianSearchInputProps> = ({
         })
       )}
       sx={{ minWidth: width }}
-      textSx={{ backgroundColor: theme.palette.background.white }}
+      textSx={{ backgroundColor: theme.palette.background.drawer }}
```

Comment:
Side quest... there was a change that cause the toolbar's clinician to show as white... should be grey like the rest of the toolbar